routes/transactionsRoute.js

```diff
@@ -0,0 +1,30 @@
+const transactionsController = require('../controllers').transactionsController;
+
+module.exports = (app) => {
+  /**
+   * @swagger
+   * /api/transactions/makeTransaction:
+   *   get:
+   *     name: MakeTransaction
+   *     summary: makes a transaction
+   *     responses:
+   *       200:
+   *         description: Successfully creates a transaction record in the Transactions table and updates portfolio.
+   */
+  app.post('/api/transactions/makeTransaction', transactionsController.makeTransaction); 
```

Comment:
I think it would be good to change route names to represent resources/nouns. Instead of '/api/transactions/makeTransaction', change to '/api/transaction'